src/main/java/com/yugabyte/sample/apps/SqlGeoPartitionedTable.java

```diff
@@ -85,6 +87,31 @@ public void createTablesIfNeeded(TableOp tableOp) throws Exception {
         LOG.info("Dropping any table(s) left from previous runs if any");
       }
 
+      // Create tablespaces if necessary.
+      if (appConfig.placementPolicies != null) {
+        appConfig.tablespaces = new String[appConfig.numPartitions];
+        for (int i = 0; i < appConfig.numPartitions; i++) {
+          // Placement policy is of the type cloud.region.zone.
+          // Split it into individual entities.
+          final String placement = appConfig.placementPolicies[i];
+          final String[] entities = placement.split("\\.");
```

Comment:
Check that we have 3 parts after split() call.

Reply:
Actually there is a check in the command line options, but I guess good to have one here too. Adding it now